docs/framework/migration-guide/mitigation-serialization-control-characters.md

```diff
@@ -10,19 +10,19 @@ ms.assetid: e065d458-a128-44f2-9f17-66af9d5be954
 ---
 # Mitigation: Serialization of control characters with the DataContractJsonSerializer
 
-Starting with the .NET Framework 4.7, the way in which control characters are serialized with the <xref:System.Runtime.Serialization.Json.DataContractJsonSerializer> has changed to conform to ECMAScript V6 and V8. 
+Starting with .NET Framework 4.7, the way in which control characters are serialized with the <xref:System.Runtime.Serialization.Json.DataContractJsonSerializer> has changed to conform to ECMAScript V6 and V8. 
  
 ## Impact
 
-In the .NET framework 4.6.2 and earlier versions, the <xref:System.Runtime.Serialization.Json.DataContractJsonSerializer> did not serialize some special control characters, such as `\b`, `\f`, and `\t`, in a way that was compatible with the ECMAScript V6 and V8 standards.
+In .NET Framework 4.6.2 and earlier versions, the <xref:System.Runtime.Serialization.Json.DataContractJsonSerializer> did not serialize some special control characters, such as `\b`, `\f`, and `\t`, in a way that was compatible with the ECMAScript V6 and V8 standards.
 
-For apps that target versions of the .NET Framework starting with the .NET Framework 4.7, serialization of these control characters is compatible with ECMAScript V6 and V8. The following APIs are affected:
+For apps that target versions of .NET Framework starting with .NET Framework 4.7, serialization of these control characters is compatible with ECMAScript V6 and V8. The following APIs are affected:
```

Comment:
```suggestion
For apps that target versions of .NET starting with .NET Framework 4.7, serialization of these control characters is compatible with ECMAScript V6 and V8. The following APIs are affected:
```

Reply:
I think this specifically meant the .NET Framework, not .NET in general.

Reply:
Interesting point. This article is specific to .NET Framework but I don't know the behavior in Core for that class.

@bartonjs, @joperezr can you help? That class does exist in Core too.

Reply:
Given this is a migration guide for Framework, should we just keep the word Framework here?

Reply:
Yes, I'll remove my suggestion.